src/openforms/templates/admin/plugin_tester.html

```diff
@@ -0,0 +1,94 @@
+{% extends 'admin/base_site.html' %}
```

Comment:
This template could probably do without (or very little) custom css if you extend from the detail template.

Also, we don't import any external CSS if possible. The Django admin has a true/false icon built in.